crates/ironclaw_gateway/static/styles/surfaces/memory.css

```
@@ -1,343 +1,355 @@
-.tree-item {
-  padding: 4px 12px 4px 16px;
-  cursor: pointer;
-  font-size: var(--text-sm);
+.search-result .snippet {
+  font-size: var(--v2-font-size-caption);
   color: var(--text-secondary);
-  display: flex;
-  align-items: center;
-  gap: 6px;
+  line-height: 1.45;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)